microbit/src/05-led-roulette/debug-it.md

```diff
@@ -45,22 +45,23 @@ $ gdb target/thumbv6m-none-eabi/debug/led-roulette
 > as long as it does not crash, you are fine.
 
 Next we will have to connect to the GDB stub. It runs on `localhost:1337` per default so in order to
-connect to it run the following:
+connect to it run the following on the gdb commandline (`(gdb)`):
```

Comment:
This is a bit confusing, I think. How about "on the `gdb` command line (that is, at the `(gdb)` prompt):"